src/main/java/cn/RecipeAPI/Models/Review.java

```diff
@@ -23,16 +23,17 @@ public class Review {
     @NotNull
     private String username;
 
-    private int rating;
+    private Integer rating;
```

Comment:
Since it is an object now it can be `null` so it could be a good idea to check it on line 32